nixos/modules/services/monitoring/ups.nix

```diff
@@ -575,6 +577,23 @@ in
       ];
     };
 
+    systemd.services.ups-killpower = {
+      enable = cfg.upsd.enable;
```

Comment:
Should we also check `cfg.upsmon.settings.POWERDOWNFLAG != null` here, so that users can easily disable this behavior?

Reply:
Maybe. Or if we want to make this configurable (and discoverable), maybe it's better to have an explicit option? power.ups.allowUpsKillPower? Or tell users to set `systemd.services.ups-killpower.enable = false`?

Is it overdesigning though? Why would someone not want to kill the UPS power? It's needed to ensure the systems come back up, so...

But I guess allowing null is an easy way to handle it -- it's not very obvious, but at least it's doable.

Data point: the apcupsd module does not have option to disable killpower. Nobody complained in 10+ years.